OpenRA.Game/Orders/UnitOrderGenerator.cs

```diff
@@ -74,7 +74,7 @@ public string GetCursor(World world, CPos xy, MouseInput mi)
 			else
 			{
 				var frozen = world.ScreenMap.FrozenActorsAt(world.RenderPlayer, mi)
-					.Where(a => a.Info.Traits.Contains<ITargetableInfo>() && !a.Footprint.All(world.ShroudObscures))
+					.Where(a => a.Info.Traits.Contains<ITargetableInfo>() && !a.Footprint.All(world.ShroudObscures) && !world.RenderPlayer.Shroud.IsUnderGapGen(a.Footprint))
```

Comment:
I don't know, it seems to work with just this code. This is to prevent having an attack cursor for hidden buildings under fog (they dont have frozen images now if gap gen is active).